app/routes/pages/components/LandingPage.js

```diff
@@ -126,6 +106,93 @@ const LandingPage = ({
         </div>
       </Flex>
 
+      <Flex className={styles.emailContainer}>
+        <Icon name="people" size={80} className={styles.emailIcon} />
+        <div className={styles.emails}>
+          <h3 className={styles.title}>Sosiale medier</h3>
+          <div>
+            Har du lyst til å følge med på hva de forskjellige delene av abakus
+            driver med? Følg oss på sosiale medier!
+          </div>
+          <Flex wrap className={styles.committeeEmails}>
+            <div className={styles.socialMediaType}>
+              <Icon name="logo-facebook" size={40} prefix="ion-" />
+              <div className={styles.socialMediaTypeLinks}>
+                {socialMedia.facebook.map((page, index) => (
+                  <a
+                    key={index}
+                    href={page.link}
+                    className={styles.socialMediaLink}
+                  >
+                    {page.textInfo}
+                  </a>
+                ))}
+                {loggedIn && (
+                  <a
+                    href="https://www.facebook.com/groups/398146436914007/"
+                    className={styles.socialMediaLink}
+                  >
+                    Gruppe for medlemmer
```

Comment:
"Gruppe for medlemmer" or "Medlemsgruppe", what sounds better?

Reply:
I agree medlemsgruppe sounds better

Reply:
Take some screenshots, and tag/send a mail to some folks in PR (and @edvardvb). They have full control on this :sunglasses: